### PR TITLE
yori: Add version 1.11

### PR DESCRIPTION
**Pull request by Restia666Ashdoll**
http://www.malsmith.net/yori/
https://github.com/malxau/yori

### CODE REVIEW

File: bucket/yori.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+            "yori.exe",
+            "Yori"
+        ]
+    ],
+    "checkver": {
+        "github": "https://github.com/malxau/yori",
+        "regex": "v([\\d.]+)"
+    },
+    "autoupdate": {
+        "url": "http://www.malsmith.net/download/?obj=yori/latest-stable/win32/ysetup.exe"
```
Comment (linsui) [inline bucket/yori.json]:
```suggestion
        "url": "http://www.malsmith.net/download/?obj=yori/$version/win32/ysetup.exe"
```

File: bucket/yori.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{
+    "version": "1.11",
+    "description": "CMD replacement shell that supports backquotes, job control, improves tab completion, file matching, aliases, command history, and more.",
+    "homepage": "http://www.malsmith.net/yori/",
+    "license": "MIT",
+    "url": "http://www.malsmith.net/download/?obj=yori/latest-stable/win32/ysetup.exe",
```
Comment (linsui) [inline bucket/yori.json]:
```suggestion
    "url": "http://www.malsmith.net/download/?obj=yori/1.11/win32/ysetup.exe",
```

**Comment by Restia666Ashdoll** [comment]
Well, those exes are just GNU tools like touch, tail and which.

**Comment by Restia666Ashdoll** [comment]
Just asking, should I add `"env_add_path": "$dir",` to the manifest ? Since, some of the functions rely on yori directory being on path(all those exe's).

**Comment by Ash258** [comment]
All of these exes are loaded to path already in yori session...

**Pull request merged**